### ISSUE TITLE
Getting Excel-Dna log messages and bitness problems

### ISSUE THREAD

**Issue by fandrei** [Excel-DNA/ExcelDna#266]
I added Excel-Dna log interception using the config file and it works for 32 bit version, but not for 64 bit. Any ideas why?

**Comment by fandrei** [comment]
I've found the reason, it's because of the ExcelDna64BitAddInSuffix option that changes the name of the assembly and so the config file doesn't apply. One possible solution is to create a duplicate of the config file before the ExcelDna packing process with the name that matches the name of the x64 assembly.
Any ideas how to fix it in a less ugly way?

**Comment by caioproiete** [comment]
@fandrei Could you expand a little more on the issue you're having and the workaround you found?

The build process creates identical copies of the `App.config` file **with the correct name** that matches all the `.xll` files generated (including the `-packed` ones), so you should have a corresponding `.xll.config` file for every single `.xll` that gets created during the build.

e.g.
![image](https://user-images.githubusercontent.com/177608/67909297-7a115f80-fb5d-11e9-91f2-7f22b8c2d5df.png)

**Comment by fandrei** [comment]
@caioproiete apparently, sometimes it doesn't. What I did to fix it - before starting the ExcelDna packing, make a copy of the "MyAddin.config" file to "MyAddin-x64.config" (and the addin assembly is named "MyAddin.dll" with "-x64" suffix in the ExcelDna64BitAddInSuffix option.

**Comment by caioproiete** [comment]
@fandrei That's odd... Not something that I've seen happening before. Every time I build a project, I see the .config files generated in the output.

If you know of any way to reproduce this behavior, let us know.

The screenshot above is from a sample project created from scratch just now, with `ExcelDna.AddIn v1.0.0`.

**Comment by caioproiete** [comment]
@fandrei I looked briefly at your sample project [`RtdMinimal`](https://github.com/fandrei/RtdMinimal/issues/1) and I suspect the issue you reported above is caused because of the same problem I saw on your RtdMinimal sample: **The name of your configuration file is not `App.config` as expected by the build system**.

I left you some [more detailed comments on an issue there](https://github.com/fandrei/RtdMinimal/issues/1).

**Comment by fandrei** [comment]
@caioproiete I see thanks.
